.github/workflows/docs.yml

```diff
@@ -0,0 +1,59 @@
+name: Generate documentation
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ["main", "publish-api-reference"]
```

Comment:
What is the "publish-api-reference" branch for?

Reply:
that's the one I was testing on. Will remove.